### ISSUE TITLE
EDMarketConnector_win_338.msi broken?

### ISSUE THREAD

**Issue by schouteg** [Marginal/EDMarketConnector#413]
When I execute EDMarketConnector_win_338.msi on my Win10 laptop, I get the following error "The system cannot open the device or file specified".
The error code is 2755.

![EDMC_error1](https://user-images.githubusercontent.com/22171448/56327927-e6f27880-617c-11e9-99ae-b1814044a18b.PNG)
![EDMC_error2](https://user-images.githubusercontent.com/22171448/56327938-efe34a00-617c-11e9-8543-4f0c5dac5389.PNG)

**Comment by schouteg** [comment]
Same issue with EDMarketConnector_win_340.msi: the installer doesn't request Admin permission.
When running the installer as admin, it installs Ok.

**Comment by StepnSteph** [comment]
Ditto. I had to open an admin CMD window to start the MSI with the proper privileges. Otherwise it would tell me that the resources existed on a network drive that didn't exist, and I don't use network drives.

I also tried the 40 MSI, which behaved the same way.

I haven't been able to update EDMC via the auto update feature so I decided to do it manually and bumped into this issue. My previous was 3.3.??. I forget specifically.

The log file generated by the installer says:

"Error 1714. The older version of Elite Dangerous Market Connector cannot be removed.  Contact your technical support group.  System Error 1612."
